### ISSUE TITLE
Total ROI is very high for a couple of channels. Whereas the optimized percentage allocation is still low

### ISSUE THREAD

**Issue by rohan1997vk** [facebookexperimental/Robyn#163]
# Contributing to FB NextGen MMM R script
I ran Robyn on my data. It has 19 channels. For a couple of those channels that ROI is way to high. While after budget allocation, the new spend share is still not high.
# Issue

A summary of what the issue is about.
![image](https://user-images.githubusercontent.com/22797824/135053526-a26706c3-1ae0-48b1-8791-598eba35cd9c.png)

# Steps to Reproduce

Please provide instructions that can be used to reproduce your issue. Usually this will include a test case that produces the wrong output.

# Expected Behavior

What you expected to happen. For example the error message you expected to see.

# Actual Results

What actually happened. For example, an error message you did not expect to see.

# Environment

R version (R --version)

**Comment by Leonelsentana** [comment]
Hi @rohan1997vk hope you are well. Have you tried decreasing channel_constr_low (range is 0.01-1) for your biggest spend channels while increasing channel_constr_up for the smaller spend channels in your budget allocator (E.g. 2 means up to 2x channels average investment from past data) ? That should help you get a better balance on the allocation.
Let us know if that helps!

**Comment by rohan1997vk** [comment]
Channel1 - (0.65,1.5) ; Historical the total spend on this channel is 0.72% - ROI is ~ 48[generated by Robyn]
Channel2 - (0.65,1.5); Historical the total spend on this channel is 3.57% - ROI is ~ 8[generated by Robyn]
Channel3 - (0.7,1.2); Historically the total spend on this channel is 46.6% - ROI is ~0 .34[generated by Robyn]

![image](https://user-images.githubusercontent.com/22797824/135066870-7a9dac9c-13de-421e-9b01-6eb93cc1527e.png)

![image](https://user-images.githubusercontent.com/22797824/135067174-d4867592-26e5-406a-bd09-ba27573d518f.png)

**Comment by gufengzhou** [comment]
Is this the chosen model? Are all Pareto models showing the same/high ROI for these channels?

**Comment by rohan1997vk** [comment]
Yes, it is one of the models. However, for all the generated models by Robyn, the end results are comparable to each other. There is no major deviations

**Comment by gufengzhou** [comment]
You can try increasing the pareto_front argument in robyn_run() to get more pareto results. Regarding the high search channels ROI, if you have some numbers in mind that you're very certain, you can use those numbers to calibrate search. That'd be the best way to deal with it.
